### PR TITLE
Get monthly stat endpoint

### PR DESCRIPTION
**Pull request by PeterOu8**
All problems are fixed, and ready to merge.

### CODE REVIEW

**Comment by PeterOu8** [comment]
image

The return response follows the given format.

File: blotztask-api/Models/MonthlyStatDTO.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+namespace BlotzTask.Models
+{
+    public class MonthlyStatDTO
+    {
+        public MonthlyStatDTO(string inputMonth) {
+            CurrentMonth = inputMonth;
+            Data = new Tasks();
+        }
+        public string CurrentMonth { get; set; }
+        public Tasks Data { get; set; }
```
Comment (sol-wizard) [inline blotztask-api/Models/MonthlyStatDTO.cs:12]:
```suggestion
        public Tasks Tasks { get; set; }
```

Data does not give any specific meaning to the object , ⬇️ readability

File: blotztask-api/Services/TaskService.cs [hunk 2]
```diff
@@ -146,5 +147,45 @@ public async Task<List<TaskItemDTO>> GetTaskByDate(DateOnly date)
             throw new Exception($"Unhandled exception: {ex.Message}");
         }
     }
+
+    public async Task<MonthlyStatDTO> GetMonthlyStats(string userId, int year, int month)
+    {
+        var input = new DateTime(year, month, 1);
+        var currentMonth = input.ToString("yyyy-MM");
```
Comment (sol-wizard) [inline blotztask-api/Services/TaskService.cs:154]: i feel like this is unnecessary, let just make currentMonth as int in the DTO unless you have a good reason for it to be string

**Review by sol-wizard — CHANGES_REQUESTED** [review]

File: blotztask-api/Services/TaskService.cs (hunk 2 above)
Comment (PeterOu8) [reply blotztask-api/Services/TaskService.cs:154]: This is meant to be a format check as I don't want any wrong format of input to break the code.

**Review by Ben0189 — DISMISSED** [review]

**Review by sol-wizard — APPROVED** [review]

**Pull request merged**